projectPage.css

```diff
@@ -117,6 +120,7 @@ img {
   }
 
   .all {
+    background-color: black;
```

Comment:
you had this as a style attribute on the body element in your html. generally bad practice to do that, so I moved it here.